tests/client/amo/components/TestCategeories.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+import { Provider } from 'react-redux';
+
+import createStore from 'amo/store';
+import Categories from 'amo/components/Categories';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+const categories = [
+  {
+    application: 'firefox',
+    name: 'motorbikes',
+    slug: 'motorbikes',
+    type: 'extension',
+  },
+  {
+    application: 'android',
+    name: 'Travel',
+    slug: 'travel',
+    type: 'extension',
+  },
+  {
+    application: 'android',
+    name: 'Games',
+    slug: 'Games',
+    type: 'extension',
+  },
+  {
+    application: 'android',
+    name: 'Cats',
+    slug: 'cats',
+    type: 'theme',
+  },
+];
+
+describe('Categories', () => {
+  function render({ ...props }) {
+    const baseProps = {
+      clientApp: 'android',
+      categories,
+    };
+    const initialState = {
+      api: { clientApp: 'android', lang: 'fr' },
+      categories: { ...categories },
+    };
+
+    return findDOMNode(findRenderedComponentWithType(renderIntoDocument(
+      <Provider store={createStore(initialState)}>
+        <Categories i18n={getFakeI18nInst()} {...baseProps} {...props} />
+      </Provider>
+    ), Categories));
+  }
+
+  it('renders Categories', () => {
+    const root = render({
+      addonType: 'extension',
+      error: false,
+      loading: false,
+    });
+
+    assert.ok(root.querySelector('.Categories-list'));
```

Comment:
Might be nice to check `assert.equal(root.textContent, 'Games\nTravel');` or something similar.